src/ansys/tools/visualization_interface/backends/pyvista/widgets/mesh_slider.py

```diff
@@ -72,10 +72,13 @@ def callback(self, state: bool) -> None:
                 self.plotter_helper._pl.scene.add_actor(actor)
         else:
             self._mb = pv.MultiBlock(self.plotter_helper._pl.scene.meshes).combine()
-            self._widget_actor = self.plotter_helper._pl.scene.add_mesh_clip_plane(self._mb)
+            show_edges = self.plotter_helper._pl._show_edges
+            self._widget_actor = self.plotter_helper._pl.scene.add_mesh_clip_plane(self._mb, show_edges=show_edges)
```

Comment:
```suggestion
            self._widget_actor = self.plotter_helper._pl.scene.add_mesh_clip_plane(self._mb, show_edges=self.plotter_helper._pl._show_edges)
```
Why not just pass the value?